content/docs/aws/deploy/existing-cluster.md

```diff
@@ -1,63 +0,0 @@
-+++
```

Comment:
Please add a redirect for this deleted file, to catch incoming links.

Add the redirect in this file:
https://github.com/kubeflow/website/blob/master/content/_redirects
See instructions at the top of the file.